### DISCUSSION TITLE
How can I get Galaxy Brain achievement?

### DISCUSSION THREAD

**Discussion by ahlyel-amine** [community/community#136815] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Select Topic Area
Question

Body
Who can help me with getting achievement of Galaxy Brain?

**Comment by oumaima-aarabe** [comment, marked as answer]
to earn the badge you need to :

Participate actively in discussions on issues, pull requests, or GitHub Discussions.
Provide thoughtful, high-quality responses or solutions that show deep understanding and insight.
Engage with the Community:

Share your knowledge and expertise in comments or discussions.
Offer valuable suggestions, code reviews, or explanations that contribute significantly to the conversation.

**Comment by oumaima-aarabe** [comment]
P.S: I don’t think it’s enabled atm. Haven’t seen it online anywhere.

**Comment by nenisrudani7** [comment]
response 1#
